test/issues/issues-test.js

```diff
@@ -252,7 +251,20 @@ describe("issues", function () {
     });
 
     if (typeof window !== "undefined") {
+
+        var throwsOnUnconfigurableProperty = false;
+        var preDescriptor = Object.getOwnPropertyDescriptor(window, "innerHeight");
+        /* eslint-disable no-restricted-syntax */
+        try {
+            Object.defineProperty(window, "innerHeight", { value: 10, configureable: true, writeable: true });
+            Object.defineProperty(window, "innerHeight", preDescriptor);
+        } catch (err) {
+            throwsOnUnconfigurableProperty = true;
+        }
+        /* eslint-enable no-restricted-syntax */
+
         describe("#1456", function () {
+            if (throwsOnUnconfigurableProperty) { return; }
```

Comment:
Any reason not to do this all inline instead of setting a single-use variable?

Also if you do the check in the `beforeEach`, you can call `this.skip();` to skip all the related tests.